doc/users/project/mission.rst

```diff
@@ -0,0 +1,28 @@
+Matplotlib Mission Statement
+============================
+
+The Matplotlib developer community develops, maintains, and supports Matplotlib
+and its extensions to provide plotting tools for the Scientific Python
+Ecosystem.  Our success is based on how effectively we enable our users, across
+scientific, industrial, commercial, and educational domain, to achieve their
+goals.
+
+We believe that a diverse developer community creates the best software, and we
+welcome anyone who shares our mission and values, as described in the code of
+conduct, to the community.
+
+
+Adapting the requirements :ref:`laid out by John Hunter
+<History>` Matplotlib should:
+
+* Support all users of the Scientific Python Ecosystem;
+* Produce high-quality raster and vector formats output;
+* Be embeddedable in graphical user interfaces for application development;
```

Comment:
Do we want to say something about web and/or interactivity? These are not our strong sides right now, but we should discuss whether that should be our goal.

Reply:
GUI application implies interactivity, though the word could be inserted to make it explicit.  I think that better interactivity in web apps like Jupyter notebook is important.  Maybe modify to "...application development, standalone and on the web."